hedera-node/hedera-mono-service/build.gradle.kts

```diff
@@ -97,6 +97,8 @@ dependencies {
 
 tasks.withType<JavaCompile> { options.annotationProcessorPath = apt }
 
+tasks.withType<Test> { maxHeapSize = "4096m" }
```

Comment:
This should be set here: https://github.com/hashgraph/hedera-services/blob/02298a25e1b5a6bfe9c2a7405e5518d366b24f37/buildSrc/src/main/kotlin/com.hedera.hashgraph.conventions.gradle.kts#L168-L172

```suggestion
```

Reply:
Updated